wagtail/admin/views/pages/history.py

```diff
@@ -1,27 +1,33 @@
 import django_filters
-from django import forms
 from django.core.exceptions import PermissionDenied
+from django.db.models import Q
 from django.shortcuts import get_object_or_404
 from django.utils.functional import cached_property
 from django.utils.translation import gettext_lazy
 
 from wagtail.admin.views.generic import history
 from wagtail.admin.views.pages.utils import get_breadcrumbs_items_for_page
+from wagtail.admin.widgets import BooleanRadioSelect
 from wagtail.models import Page, PageLogEntry
 from wagtail.permissions import page_permission_policy
 
 
 class PageHistoryFilterSet(history.HistoryFilterSet):
-    hide_commenting_actions = django_filters.BooleanFilter(
-        label=gettext_lazy("Hide commenting actions"),
-        method="filter_hide_commenting_actions",
-        widget=forms.CheckboxInput,
+    is_commenting_action = django_filters.BooleanFilter(
+        label=gettext_lazy("Is commenting action"),
+        method="filter_is_commenting_action",
+        widget=BooleanRadioSelect,
```

Comment:
Not quite sure about the naming, happy to take suggestions.

Also, I'd like to clean up how we do boolean filters. I think we're better off with a `ChoiceFilter` and a plain `RadioSelect` with predefined choices. `BooleanRadioSelect` does a bit too much handling around the boolean vs boolean-ish strings. I'll look into this separately, as I'm afraid this PR is getting big enough as it is.